Spring-Boot/Spring-Boot-Academy/src/test/java/tw/waterball/judgegirl/springboot/academy/controllers/ExamControllerTest.java

```diff
@@ -599,27 +596,36 @@ void testGetStudentExamProgressOverview() throws Exception {
         assertEquals(secondQuestion.getProblemTitle(), otherProblem.getTitle());
     }
 
+    @DisplayName("Give an exam with three questions A,B,C and the problem related to B has been deleted,"
+            + "When get exam overview,"
+            + "question A should have problem title and problem be not archived and"
+            + "question B should be notFound and"
```

Comment:
In the testcase￼￼￼, the questions have be named with q1, q2, q3, but you have questions A,B,C in the display name. Please use the same naming method.